.circleci/config.yml

```diff
@@ -216,6 +216,51 @@ jobs:
           environment:
             <<: *github_team_name_slug
 
+  test_staging:
+    <<: *test_container_config
+    working_directory: ~/staging
+    steps:
+      - checkout
```

Comment:
It's possibly a little strange to be checking out at this point (this command does a number of things including cloning the repo that the job is configured to build i.e offender-allocation-manager). You will see that I clone the integration tests repo into this folder. The reason for using `checkout` is that you get other functionality thrown in including adding Github to known-hosts. If we did not do this we would have to add Github to known hosts manually. Happy to change if this feels to messy

Reply:
Perfectly fine to be checking out other repos though.